README.md

```diff
@@ -79,6 +79,12 @@ If you are running inside Docker you will want to enter a shell in the `web` con
 Tests are written with pytest; the current examples should show uses of test
 client, fixtures, request mocking, settings override, and so on.
 
+### Using the shell
+
+You can get autocomplete, vi-mode and other features if you use 
+[ptpython](https://github.com/prompt-toolkit/ptpython) as your shell
+via `./manage.py shell -i ptpython`.
```

Comment:
It changes the default, you'd want to insert it at the end if you wanted it to be optional, I assume. I don't really mind either way, might as well be the default, so this text could therefore be updated.

I guess in a bigger project, if we wanted this to be per-user and not have custom shells in the core code (if one person wanted ipython, another bpython, etc) - perhaps you'd want ptpython to be installed for a particular user (`brew install ptpython`, `pip install --user`, whatever) and then have some way of using that without having to change the code. Probably not worth worrying about :)